azure/deploy_azure.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+# Deploy Azure instance using terraform
+
+# Export environment variables
+source .env
```

Comment:
What does this do?

Reply:
Just an assertion to seee if you're paying attention, haha.

Reply:
ha